.github/no-response.yml

```diff
@@ -0,0 +1,16 @@
+# ProBot No Response Bot
+# https://probot.github.io/apps/no-response/
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+
+# Label requiring a response
+responseRequiredLabel: 'Needed: more information'
+
+# Comment to post when closing an Issue for lack of response. Set to `false` to disable
+closeComment: >
+  This issue has been automatically closed because there has been no response
+  to our request for more information from the original author. With only the
+  information that is currently in the issue, we don't have enough information
+  to take action. Please reach out if you have or find the answers we need so
+  that we can investigate further. Thanks!
```

Comment:
Maybe we could add a link to our docs here, I use this response for these cases

Closing this as the user [hasn't responded or given more information](https://docs.readthedocs.io/en/latest/contribute.html#initial-triage), feel free to reopen with more information if you are still having this problem :).